packages/react-core/src/components/DataList/DataList.tsx

```diff
@@ -3,6 +3,12 @@ import * as React from 'react';
 import { css } from '@patternfly/react-styles';
 import styles from '@patternfly/react-styles/css/components/DataList/data-list';
 
+export enum WrapModifier {
```

Comment:
Would be good to prefix to `DataListWrapModifier`, otherwise it could clash if there is another WrapModifier exported somewhere else

Reply:
You could also get rid of the enum altogether if you'd like since most of our consumers use our types. It's up to you.